pkg/server/helpers.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/docker/pkg/stringid"
+	"google.golang.org/grpc"
+
+	"github.com/containerd/containerd"
+
+	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
```

Comment:
Ack. I think splitting them into three groups (i.e., standard, others, cri-containerd) follows the k8s convention better, but I am fine wit this.